civihr_employee_portal/views/views_export/views_emergency_contacts.inc

```diff
@@ -138,9 +156,10 @@ $translatables['emergency_contacts'] = array(
   t('Address'),
   t('Relationship with Employee'),
   t('Notes'),
+  t('<!-- This field is totally empty, but uses markup from civihr-employee-portal-theme to display the button. See civihr_default_theme/templates/view/views-view-field--emergency-contacts--non-dependant-emergency-contact--emergency_contact_deletion.tpl.php -->'),
+  t('All'),
   t('Non Dependant Emergency Contacts'),
   t('A list of emergency contacts where the "dependant" flag is false'),
-  t('non_dependant_emergency_contact'),
```

Comment:
That's what the auto exporter tells me, it's been saying that for the last few exports but I ignored it until now. I don't know why it was in the translatables in the first place since it's just the display ID